clippy_lints/src/lib.rs

```diff
@@ -1222,8 +1224,8 @@ pub fn register_plugins(store: &mut rustc_lint::LintStore, sess: &Session, conf:
     store.register_late_pass(|| box strings::StrToString);
     store.register_late_pass(|| box strings::StringToString);
     store.register_late_pass(|| box zero_sized_map_values::ZeroSizedMapValues);
-    store.register_late_pass(|| box vec_init_then_push::VecInitThenPush::default());
     store.register_late_pass(move || box types::PtrAsPtr::new(msrv));
+    store.register_late_pass(|| box vec_init_then_push::VecInitThenPush::default());
```

Comment:
You'll have to add
```suggestion
    store.register_late_pass(|| box vec_init_then_push::VecInitThenPush::default());
    store.register_late_pass(move || box types::PtrAsPtr::new(msrv));
    store.register_late_pass(|| box redundant_slicing::RedundantSlicing);
```

I think you had this before, but it disappeared during rebase?

Reply:
That was a bad rebase. Fixed now.